tests/30-rdfc-interop.js

```diff
@@ -20,15 +20,15 @@ const {
   match: verifierMatches
 } = endpoints.filterByTag({tags: [...tags], property: 'verifiers'});
 
-describe('ecdsa-rdfc-2019 (interop)', function() {
+// skip these tests if local true
```

Comment:
```suggestion
```

Nit: i dont think this comment is needed

Reply:
It may help to know when those are run, but I do wonder about spelling out the variable rather than "local true". But...definitely a nit either way.